playground/src/pages/_app.tsx

```diff
@@ -10,6 +10,31 @@ export default function App({ Component, pageProps }: AppProps) {
         <link rel="icon" href="/logo.png" sizes="any" />
       </Head>
       <Component {...pageProps} />
+      <footer className="p-4 text-white border-t border-opacity-10 text-xs">
```

Comment:
I think it is a bit too white:
```suggestion
      <footer className="p-4 text-gray-400 border-t border-gray-800 text-xs">
```